model/import/Repository/TemplateRepositoryInterface.php

```diff
@@ -30,37 +32,48 @@ interface TemplateRepositoryInterface
     public const DEFAULT_DEFINITION = [
         'name' => [
             'header' => 'required',
-            'value' => 'string'
+            'value' => 'string',
         ],
         'question' => [
             'header' => 'required',
-            'value' => 'string'
+            'value' => 'string',
         ],
         'shuffle' => [
             'header' => 'optional',
-            'value' => 'boolean'
+            'value' => 'boolean',
+            'default' => 'false',
+        ],
+        'language' => [
```

Comment:
I think in this template specifications we could have column identifiers to help us to `guess` how to handle those values properly.

Example:

```
        'question' => [
            'reference' => 'question_title',
            'header' => 'required',
            'value' => 'string',
        ],
        'min_choices' => [
            'reference' => 'question_max_choices',
            'header' => 'optional',
            'value' => 'integer',
            'default' => 0,
        ],
```

So in this way, in the implementation even if we do not know the name used in the CSV we know the internal reference, so we can validate it properly. Wdyt?